service/controller/v5/arm_templates/main.json

```diff
@@ -68,6 +68,10 @@
       "type":"string",
       "defaultValue":"dns_cname_setup.json"
     },
+    "ignitionContainerSetupTemplateFile":{
+      "type":"string",
+      "defaultValue":"ignition_container_setup.json"
+    },
```

Comment:
I need some more explanation on why we need this container, what it does and how all this works. Do we have specs or docs or some diagrams for that?

Reply:
https://github.com/giantswarm/giantswarm/issues/2816#issuecomment-430650737